app/Observers/ProjectObserver.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\Stat;
+
+class ProjectObserver
+{
+    /**
+     * Handle the Stat "created" event.
+     *
+     * @param  \App\Models\Stat  $Stat
+     * @return void
+     */
+    public function created(Stat $Stat)
```

Comment:
You are observing Project created event but why you are model binding with Stat?
I am a bit confused here.